gcloud/storage/blob.py

```diff
@@ -358,7 +402,8 @@ def _check_response_error(request, http_response):
             raise make_exception(faux_response, http_response.content,
                                  error_info=request.url)
 
-    def upload_from_file(self, file_obj, rewind=False, size=None,
+    # pylint: disable=too-many-locals
```

Comment:
Yea, can we make a ticket to deal with this by breaking the method up into more manageable sized pieces?